src/configupdater/configupdater.py

```diff
@@ -393,6 +393,15 @@ def set(self, option, value=None):
             self.__setitem__(option, value)
         return self
 
+    def items(self):
+        """Return a list of (name, value) tuples for each option in
+        this section.
+
+        Returns:
+            list: list of :class:`Section` or :class:`Option` objects
```

Comment:
This is the wrong return type. It's a list of tuples.

Reply:
Meh. Copy/paste ... I'll fix that.